oktoberfest/predict/predictor.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import annotations
+
+import importlib.util
+import logging
+from pathlib import Path
+from typing import Dict, List
+
+import anndata
+import numpy as np
+import pandas as pd
+
+from ..data.spectra import FragmentType, Spectra
+from ..utils import Config, group_iterator
+from .alignment import _alignment, _prepare_alignment_df
+from .dlomix import DLomix
```

Comment:
does this always work? I see you have set it as an optional dependency in `pyproject.toml`, which is nice, and then you also check downstream in the `from_config` method to see if `dlomix` is available. But the import is most definitely failing if dlomix is not installed.